### PR TITLE
[gitpod] added basic .gitpod.yml

### PR DESCRIPTION
**Pull request by JanKoehnlein**
An initial gitpod config. Pre-builds should be enabled.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tasks:
+- init: ./gradlew build
```
Comment (cdietrich) [inline .gitpod.yml]: any reason not to call  `./1-gradle-build.sh`

**Comment by cdietrich** [comment]
@kthoms @Hejado any idea why https://ci.eclipse.org/xtext/job/xtext-core/job/jk%252Fgitpod/
does not build? bad branch name?

**Comment by cdietrich** [comment]
`12:37:59  ERROR: Labels must follow required specs - https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set: xtext-core-jk/gitpod-4
12:37:59  Finished: FAILURE`

**Comment by cdietrich** [comment]
```
12:37:59  ERROR: Labels must follow required specs - https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set: xtext-core-jk/gitpod-4
12:37:59  Finished: FAILURE
```

File: .gitpod.yml (hunk 1 above)
Comment (JanKoehnlein) [reply .gitpod.yml]: Nope, didn't know it existed. Just followed the instructions from `README.MD`.

**Comment by cdietrich** [comment]
created https://github.com/eclipse/xtext/issues/1595 about the label calculation

**Comment by JanKoehnlein** [comment]
I used `jk/gitpod` as a branch name. Is it no longer the default to use a scheme `<user>/<name>`?

**Comment by cdietrich** [comment]
the deault scheme is `<user>_<name>`
looks like your the first one using `/` since we moved to JIRO

but it should work with `/ ` as well. 
thus i created the issue to look at it.

**Comment by LorenzoBettini** [comment]
I've just tried that and it seems to work apart a few compilation errors :)
actually it's the first time I'm using Gitpod and I can't seem to find a way to run unit tests... is that intentional?

**Comment by cdietrich** [comment]
it also looks like we dont have a gradle.properties that sets 
`org.gradle.jvmargs = -Xmx2G`

**Comment by cdietrich** [comment]
@LorenzoBettini to me it looks like java language server does not support project dependency substitution. it uses dependencies from maven central (M2) instead

**Comment by cdietrich** [comment]
dependencies

**Comment by LorenzoBettini** [comment]
> @LorenzoBettini to me it looks like java language server does not support project dependency substitution. it uses dependencies from maven central (M2) instead

I'm not sure I understand what you mean by "project dependency substitution"... is that related to compilation error or to the fact I cannot run JUnit tests?

**Comment by cdietrich** [comment]
no i see compile errors.

**Comment by LorenzoBettini** [comment]
and concerning JUnit tests it looks like that's not implemented yet: https://github.com/eclipse-theia/theia/issues/3003

**Comment by JanKoehnlein** [comment]
I am inclined to revert this. Gradle's Xtend build and the Java LS seem to cause too many synchronization problems.

**Comment by szarnekow** [comment]
I'm also in favour of making this available when it's more robust. The current state looks promising, but it's not quite there yet.

**Pull request merged**